scripts/check_compliance.py

```diff
@@ -223,6 +217,23 @@ def run(self):
         self.check_no_undef_within_kconfig(kconf)
         self.check_top_menu_not_too_long(kconf)
 
+    def get_modules(self, modules_file):
+        """
+        Get a list of modules and put them in a file that is parsed by
+        Kconfig
+
+        This is needed to complete Kconfig sanity tests.
+
+        """
+        try:
+            modules = sh.west("list", "--format={posixpath}", **sh_special_args)
+        except sh.CommandNotFound as e:
+            self.skip("Command not found: " + str(e))
+        except sh.ErrorReturnCode as e:
+            self.skip(e)
+
+        create_modules_file(modules, modules_file)
```

Comment:
Could inline this function here (and rename `get_modules()` to `create_modules_file()`). Doesn't seem to be any other users.

`get_*()` makes it sound like it returns a value to me, but I know this is old code.